guide/getting-started/getting-started.md

```diff
@@ -34,70 +34,68 @@ Illustration sources
 
 # Getting started
 
-Welcome to the Bitcoin Design Guide! You’re in for an adventure. This chapter is all about the big concepts that are helpful to be familiar with when creating Bitcoin products. You’ll find a summary of concepts on this page, click through if you would like to dig deeper into a specific topic. The guide isn't a comprehensive overview of all Bitcoin topics, but only the ones you should know as a designer working in this space.
-
 ---
 
 ### [Why Bitcoin is unique]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})
 
-The internet has opened an infinite design space for information and communication. Bitcoin adds the new dimension of transferring value that could bring similar change as previous waves of innovation.
+Familiarize yourself with some of the most significant concepts that bitcoin designers need to know.
```

Comment:
I preferred the original text, as it already gave a brief summary of the content. The updated text only teases but doesn't provide any real information.

Reply:
Reverted in https://github.com/BitcoinDesign/Guide/pull/221/commits/7aaf56ee1b2506f5293704ee23824570ab259633

Reply:
I think there’s an issue with the original where it’s not immediately clear what the copy is actually describing. I think it could be read as grandiose.

Reply:
@sqcken will re-open this once we get a clear picture on what stuff are actually describing. Will re-open when we're done with ironing out other pages.